.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,51 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+1. Consider creating this PR as draft, for early feedback ping code owners
+2. Structure your PR accordingly to the focus areas, e.g. `api` containing all exposed API specs
+3. Write meaningful commit messages and PR description
+
+For more details, please check the [CONTRIBUTING](../CONTRIBUTING.md) guide.
+-->
+
+### What this PR does
+**Before this PR:**
+
+**After this PR:**
+
+Jira: <!-- optional: link to Jira issue -->
+Link to demo recording: <!-- optional: link to a demo recording -->
+
+### Breadcrumbs
+The following tradeoffs were made:
+
+The following alternatives were considered:
+
+Links to places where the discussion took place: <!-- optional: slack, GH issue, mailinglist, ... -->
+
+### Special notes for your reviewer
+
+<!-- optional -->
+
+### Checklist
+
+This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
+Approvers are expected to review this list.
+
+- [ ] Design: An [ADR](https://drive.google.com/drive/folders/1zr2gubRU-asXjZSFZvKmcuGOV4jlUGV-) or [DDR](https://drive.google.com/drive/folders/1L9R0mtcl-5oRc9dzRdcw4ZGVJLtj-14o?usp=drive_link) was considered and is present (link) or not required
+- [ ] PR: The PR description is expressive enough and will help future contributors
+- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
+- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
+- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
+- [ ] Deployment: The deployemnt process was considered and addressed if required
+- [ ] Testing: New code requires new unit tests.
+- [ ] Documentation: Is the documentation updated? Either in the doc located in focus area, in the README or in the code itself.
+- [ ] Customers: Is this change affecting customers? Is the release plan considered?
+
+### Release note
```

Comment:
Do we use this field in a release process?

Reply:
Not now, which means it can be added later on.